eZ/Bundle/EzPublishRestBundle/Resources/config/services.yml

```diff
@@ -420,7 +418,7 @@ services:
             - { name: ezpublish_rest.input.handler, format: xml }
 
     ezpublish_rest.templated_router:
-        class: "%ezpublish_rest.templated_router.class%"
+        class: Symfony\Bundle\FrameworkBundle\Routing\Router
```

Comment:
There's one thing I have doubt about, but maybe it's not important.

Before #1905 was merged, `%router.class%` (see value of `%ezpublish_rest.templated_router.class%` above) was overriden [there](https://github.com/ezsystems/ezpublish-kernel/blob/v6.7.1/eZ/Bundle/EzPublishCoreBundle/Resources/config/routing.yml#L3), so shouldn't we use here also `eZ\Bundle\EzPublishCoreBundle\Routing\DefaultRouter` instead of `Symfony\Bundle\FrameworkBundle\Routing\Router`?